src/sentry/auth/providers/fly/provider.py

```diff
@@ -45,19 +45,13 @@ def get_refresh_token_url(self):
         return ACCESS_TOKEN_URL
 
     @classmethod
-    def build_config(self, state: Any, organization: Optional[Any] = None):
+    def build_config(self, resource: Optional[Any] = None):
         """
         On configuration, we determine which provider organization to configure sentry SSO for.
         This configuration is then stored and passed into the pipeline instances during SSO
         to determine whether the Auth'd user has the appropriate access to the provider org
         """
-        org = organization
-        if not organization:
```

Comment:
Are we sure this is safe to take out? Is it possible for `id` to not be in resource? And if so, how do we handle that case with the new code?

Reply:
Ah yea

So this isn't being utilized anywhere atm.
Since this is specific per provider, we can shape this `build_config` however we want per provider.
for Fly in particular, their organization resource will always have an 'id'